README.md

```diff
@@ -82,3 +82,53 @@ Your implementation should satisfy the following reqirements:
  * If `min_bucket_count > max_bucket_count`, either raise an error, or consider a reasonable way to decide which values to use instead
  * If there are fewer products than `min_bucket_count`, just return a single bucket. Consider what the `to` and `from` values should be
 * You will likely need a way to compare two "sets" of buckets, i.e. buckets having different `interval` values. Explain why you chose the one you made
+
+
+## Creating Elasticsearch synonyms file
+Elasticsearch gives the oppotunity for providing an synonym file, consisting of words that are connected and should evaluated to each other during searcing. For example, if a user types "ps4" we want to also search for "playstation 4". 
+
+We represent a synonym as a dictionary consisting of two text strings and a synonym type: `{'text1': 'ps4', 'text2': 'playstation 4', 'synonym_type': 'T'}`
+
+* `synonym_type` can be one of the two types:
+ * One-way, represented by `'O'`: `text1` evaluate to `text2` but not the other way around. E.g.: `'Animal' -> 'Cat'` 
+ * Two-way, represented by `'T'`: Each of the text strings `text1` and `text1` evaluate to the other. E.g.: `'wow' <-> 'World Of Warcraft'`
+
+Example data are found in `synonyms_data.py`.
+
+We want these synonym dictionaries to be outputted in a text file understandable by Elasticsearch in the following format:
```

Comment:
Does this block describe the desired output, or a description of the task? It's not clear to me.